Library/Homebrew/extend/os/mac/utils/bottles.rb

```diff
@@ -16,8 +16,13 @@ class Collector
       alias generic_find_matching_tag find_matching_tag
 
       def find_matching_tag(tag)
-        generic_find_matching_tag(tag) ||
-          find_older_compatible_tag(tag)
+        # Used primarily by developers testing beta macOS releases.
+        if ARGV.skip_or_later_bottles?
```

Comment:
Could this be gated specifically to `OS::Mac.prerelease?`?

Reply:
Updated, and updated the spec as well.